content/news/2018-11-16-the-navcoin-community-fund-is-live.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: news
+generated_by_cms: true
+title: The NavCoin Community Fund is live
+author: Kieren Hyland
+date: '2018-11-16T11:00:18+13:00'
+feature_image: /images/uploads/community-fund-live.png
+categories:
+  - Nav News
+---
+The latest updates to the NavCoin Core protocol have activated, and as a result, the NavCoin Community Fund is ready to accept your proposal for consideration and funding by the network. 
+
+The latest updates to the NavCoin Core protocol have activated, and as a result, the NavCoin Community Fund is ready to accept your proposal for consideration and funding by the network. 
+
+### A world first!
+
+Delivering a major upgrade like the Community Fund is a significant achievement - both for the Core dev team and all the other community members who contributed their time and skill to bring it to life. 
+
+In case you aren’t already aware, the NavCoin Community fund [is the world’s first and only fully-decentralised fund](https://navcoin.org/en/notices/2018-10-24-navcoin-core-4-4-0/). Both in the way it’s governed and how the funds are managed. 
+
+* It’s not controlled or managed by any individual or single group, and the funds are generated by the network itself. Now that it’s going, nothing can stop it.
+* Decentralization is also achieved by a unique dual-voting format, where the first vote is to accept proposals, and the second is to release funds at milestones or completion - and yours could be among them!
+
+### What’s next?
+
+Well, that’s totally up to you. This is your Community Fund. There are no rules to follow for what your project should or shouldn’t be. All you have to do is convince NavCoin stakers that your project is worthy of their vote, then follow through and deliver it as promised. 
+
+There are two easy ways to submit your project:
+
+* Download the [NavCoin NEXT wallet](http://next.navcommunity.net/) and take advantage of its built-in proposal form.
+* Go to NavCoin.org and use the proposal tool on [the Community Fund page](https://navcoin.org/en/community-fund/) (located at step 4 of the creation guide). This will generate a piece of code that you paste into your NavCoin Core Wallet.
+
+For the full instructions on how to submit a proposal - [please view this guide in the Knowledge Base](https://info.navcoin.org/knowledge-base/creating-a-community-fund-proposal/).
+
+### Viewing Proposals
+
+You can view proposals once they've been submitted to the blockchain using the [NavCommunity Proposal Viewer](https://communityfund.nav.community/) or alternatively on the [NavExplorer website](https://www.navexplorer.com/community-fund/proposals/pending). 
```

Comment:
https://communityfund.nav.community/ is not a simple proposal viewer but a discussion platform.